Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -2540,4 +2540,82 @@ final class DeclarationTests: XCTestCase {
       fixedSource: "let foo: [Int] = []"
     )
   }
+
+  func testInitAccessorsWithDefaultValues() {
+    assertParse(
+      """
+      struct Test {
+        var pair: (Int, Int) = (42, 0) {
+          init(initialValue) {}
+
+          get { (0, 42) }
+          set { }
+        }
+      }
+      """
+    )
+
+    assertParse(
+      """
+      struct Test {
+        var pair: (Int, Int) = (42, 0) {
+          init initializes(a) {}
+
+          get { (0, 42) }
+          set { }
+        }
+      }
+      """
+    )
+
+    assertParse(
+      """
+      struct Test {
+        var pair: (Int, Int) = (42, 0) {
+          get { (0, 42) }
+          set { }
+
+          init(initialValue1️⃣) {}
+        }
+      }
+      """,
+      substructure: Syntax(
+        InitializerDeclSyntax(
+          initKeyword: .keyword(.`init`, leadingTrivia: [.newlines(2), .spaces(4)]),
```

Comment:
Not super important but you don’t need to specify `, leadingTrivia: [.newlines(2), .spaces(4)]` here.

Reply:
No worries, fixed!